packages/monitoring-system/data-processor/package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Data ingestion and transformation system for GitHub Repo Automation Bots",
   "scripts": {
     "compile": "tsc -p .",
+    "clean-compile": "rm -rf ./build; rm -rf ./node_modules; npm install && npm run compile",
     "pretest": "npm run compile",
     "fix": "gts fix",
     "lint": "gts check",
-    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit ./build/test",
+    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit --recursive ./build/test/unit",
```

Comment:
Moved tests to `test/unit` because I'll be adding integration tests under `test/integration`